styles/components/_form.scss

```diff
@@ -26,7 +26,7 @@ textarea,
   color: inherit;
   border-radius: 3px;
   font: {
-    family: "Helvetica Neue", Helvetica, Arial, sans-serif;
+    family: "Noto Sans", sans-serif;
```

Comment:
You're right, I'm looking at it, it looks like the user agent has higher specificity.